Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -486,11 +561,106 @@ def departingemployee_remove_command(client, args):
     if user_id:
         de_context = {"UserID": user_id, "Username": username}
         readable_outputs = tableToMarkdown(
-            f"Code42 Departing Employee List User Removed", de_context
+            "Code42 Departing Employee List User Removed", de_context
         )
         return readable_outputs, {"Code42.DepartingEmployee": de_context}, user_id
     else:
-        return_error(message="Could not remove user from Departing Employee List")
+        return_error(message="Could not remove user from the Departing Employee List")
+
+
+@logger
+def departingemployee_get_all_command(client, args):
+    employees = client.get_all_departing_employees()
+    if employees:
+        employees_context = [
+            {
+                "UserID": e["userId"],
+                "Username": e["userName"],
+                "DepartureDate": e.get("departureDate"),
+                "Note": e["notes"],
+            }
+            for e in employees
+        ]
+        readable_outputs = tableToMarkdown("All Departing Employees", employees_context)
+        return (
+            readable_outputs,
+            {"Code42.DepartingEmployee(val.UserID && val.UserID == obj.UserID)": employees_context},
+            employees,
+        )
+    else:
+        return_error(message="Could not get all Departing Employees")
```

Comment:
could we move these error messages closer to where the request gets made an include the exception that is thrown there in the message detail?

Reply:
From looking at the coding conventions, it seems kind of unnecessary to return `None` from the client. Not sure why it was designed like that. I'll fix it, I want those errors to propagate up to `return_error`, that's even how their example does it.